geojson2svg/geojson2svg.go

```diff
@@ -90,11 +93,12 @@ func (svg *SVG) Draw(width, height float64, opts ...Option) string {
 // DrawWithProjection renders the final SVG with the given options to a string.
 // All coordinates will be converted by the given projection, then scaled to fit into the svg.
 func (svg *SVG) DrawWithProjection(width, height float64, projection ScaleFunc, opts ...Option) string {
+
 	for _, o := range opts {
 		o(svg)
 	}
 
-	sf := makeScaleFunc(width, height, svg.padding, svg.points(), projection)
+	sf := svg.makeScaleFunc(width, height, svg.padding, svg.points(), projection)
```

Comment:
do we need to pass through `svg.padding` and `svg.points()` since `makeScaleFunc` is already being called on `svg`?